### ISSUE TITLE
create docs

### ISSUE THREAD

**Issue by bourdakos1** [cloud-annotations/cloud-annotations#168]
## Guides
- [x] Getting started
- [x] Preparing training data
- [x] Object detection
- [x] Classification
  - [x] Upload videos/images
  - [x] How to create a label
  - [x] How to label
  - [ ] how to label negatives
- [x] Labeling with a team
  - [x] how to share a bucket
  - [ ] show ui with multiple people editing
- [x] Uploading images/labels via API
  - [x] Go over COS API
  - [x] maybe talk about the s3 sdks / cli
- [ ] Exporting annotations via GUI
  - [ ] info on Pascal VOC
  - [ ] info on Create ML / Turi Create
  - [ ] info on YOLO
  - [ ] show dropdown UI
- [ ] Exporting annotations via API
  - [ ] only supports Cloud Annotations format
  - [ ] go over format
  - [ ] maybe show how to convert
- [ ] Training overview
  - [ ] create a WML instance
- [x] Training via GUI
  - [x] click train
  - [x] choose a WML instance
  - [ ] talk about training UI / what is loss etc...
  - [ ] mention there are more training options via CLI
- [x] Installing the Cloud Annotations CLI (cacli)
  - [x] homebrew
  - [x] curl
  - [x] download the binary
- [ ] Training via CLI
  - [ ] how to login
  - [ ] go over various training options
- [ ] Training with Google Colab
  - [x] Exporting annotations
  - [x] Uploading to Google Drive
  - [ ] Using Google Colab
- [ ] Non-interactive training (useful for CI)
  - [ ] getting WML credentials
  - [ ] getting COS HMAC credentials
  - [ ] show how to training using this info and bucket so it doesn't ask for more info
- [ ] Custom training
  - [ ] You can use a custom training zip
  - [ ] Show example using the TensorFlow object detection API
- [x] Downloading a model via GUI
  - [x] click the download button
  - [ ] this downloads EVERYTHING (tensorflow, tflite, core ml, tfjs) SLOW
- [x] Downloading a model via CLI
  - [x] run cacli download model-id
  - [ ] you can add flags to only download certain models (--coreml) much faster
- [x] Using a model
  - [x] point to all the demos
- [ ] Auto labeling
  - [ ] once your initial model is trained Cloud Annotations can use it to help label more images
  - [ ] show how the UI works

**Comment by bourdakos1** [comment]
There should be documentation on:
- how to use the online tool
- how to use the CLI
- how to use the sdks
  -  javascript
  -  (swift)
  -  (java)
  -  (python)

**Comment by bourdakos1** [comment]
- CLI: none
- js object detection sdk:

**Comment by bourdakos1** [comment]
Guides:
- Getting started
  - Cloud Annotations overview
  - Sign up for IBM Cloud
- Preparing training data
  - training data best practices
  - log in to Cloud Annotations
  - Create COS instance
  - Create a bucket/project
  - Classification vs object detection
- Object detection
  - Upload videos/images
  - How to create a label
  - How to label
  - how to label negatives 
- Classification
  - Upload videos/images
  - How to create a label
  - How to label
  - how to label negatives
- Labeling with a team
  - how to share a bucket
  - show ui with multiple people editing
- Uploading images/labels via API
  - Go over COS API
  - maybe talk about the s3 sdks / cli
- Exporting annotations via GUI
  - info on Pascal VOC
  - info on Create ML / Turi Create
  - info on YOLO
  - show dropdown UI
- Exporting annotations via API
  - only supports Cloud Annotations format
  - go over format
  - maybe show how to convert
- Training overview
  - create a WML instance
- Training via GUI
  - click train
  - choose a WML instance
  - talk about training UI / what is loss etc...
  - mention there are more training options via CLI
- Training via CLI
  - how to login
  - go over various training options
- Non-interactive training (useful for CI)
  - getting WML credentials
  - getting COS HMAC credentials
  - show how to training using this info and bucket so it doesn't ask for more info
- Custom training
  - You can use a custom training zip
  - Show example using the TensorFlow object detection API
- Downloading a model via GUI
  - click the download button
  - this downloads EVERYTHING (tensorflow, tflite, core ml, tfjs) SLOW
- Downloading a model via CLI
  - run cacli download model-id
  - you can add flags to only download certain models (--coreml) much faster
- Using a model
  - point to all the demos
- Auto labeling
  - once your initial model is trained Cloud Annotations can use it to help label more images
  - show how the UI works
